code/modules/projectiles/projectile/bullets/special.dm

```diff
@@ -25,3 +25,33 @@
 	if(iscarbon(target))
 		var/mob/living/carbon/M = target
 		M.silent = max(M.silent, 10)
+
+// Special HoS "Bullets"
+
+/obj/item/projectile/bullet/hos
+	name = "3D printed .454 round"
+	damage = 25
+
+/obj/item/projectile/bullet/hos/hv
+	name = "3D printed .454 HV round"
+	speed = 0.2
+
+/obj/item/projectile/bullet/hos/heavy
+	name = "3D printed .454 heavy round"
+	damage = 35
+	knockdown = 50
+
+/obj/item/projectile/bullet/hos/heavy/on_hit(atom/target, blocked = FALSE)

```

Comment:
due to how even minor throws into walls stun right now, please remove this bit. even shotgun slugs dont do this